chess-AI/train.py

```diff
@@ -249,19 +227,18 @@ def main():
     device = torch.device('cuda:0' if torch.cuda.is_available() else 'cpu')
 
     # Get dataset parameters from dashboard
-    stocktrain_games, stocktrain_epochs = 1000,10
-    mcts_games, mcts_epochs = 100,100
-    batch_size, learning_rate = 8,0.1
+    stocktrain_games, stocktrain_epochs = 1000, 10
+    mcts_games, mcts_epochs = 100, 100
+    batch_size, learning_rate = 8, 0.1
 
     # Gets stockfish training object, and sets parameters (elo,depth)
-    stockfish_path = "C:/Users/juddb/Stockfish/stockfish_14.1_win_x64_avx2/stockfish_14.1_win_x64_avx2.exe"
+    stockfish_path = r"C:\Users\jhoyo\Downloads\stockfish_14.1_win_x64_avx2.exe"
```

Comment:
Doesn't need to be part of this PR, but can all the Windows people @JuddBE @fshcat move the stockfish binary to "../../chess-engine/stockfish_14.1_win_x64_avx2.exe" so that the path doesn't change between PRs? As part of that change, we can update this initialization to check the system and set the stockfish path accordingly (like we do in [`create_stockfish_wrapper`](https://github.com/ucfai/knightros-gambit/blob/master/firmware/raspi/util.py#L38)). I can add that in a new PR, just wanted to let you guys know so that you can move the binary to the right place.

Reply:
I'll do the initialization thing after this PR is merged, but for now can you just update this line to reflect the path I put in the comment above?